lib/ruby_parameter_store/retrieve.rb

```diff
@@ -0,0 +1,59 @@
+module RubyParameterStore
+  class Retrieve
+    require 'singleton'
+
+    include Singleton
+
+    def self.get_parameter(name)
+      if name.is_a? String
+        params[name.to_sym]
+      else
+        params[name]
+      end
+    end
+
+    def self.must_get_parameter(name)
```

Comment:
As we discussed, let's change this to get! or something.

Reply:
made that change.  also added a `clear` method to remove memoized param values and force a call out to AWS.